lib/runBlock.js

```diff
@@ -83,7 +83,7 @@ module.exports = function (opts, cb) {
     async.eachSeries(block.transactions, processTx, cb)
 
     function processTx (tx, cb) {
-      var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).cmp(new BN(tx.gasLimit).add(gasUsed)) === -1
+      var gasLimitIsHigherThanBlock = new BN(block.header.gasLimit).lt(new BN(tx.gasLimit).add(gasUsed))
```

Comment:
Not sure, what the semantics SHOULD be here, but this also evaluates to ``true`` on equal case, which is not what the variable name ``gasLimitIsHigherThanBlock`` is saying (same as in the old version). Refactoring generally is ok. Would also switch this to a ``gte`` (or ``gt`` ?) comparison for better readability.